terminal-demo-app/src/MainPage.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 
-import Backend from "./backend";
+import Backend from "./client";
```

Comment:
doesn't this have to be import Client and then in the code it has to change to new Client instead of new Backend()

Reply:
It doesn't technically have to, because './client' exports a default class. 

i.e. if ./client only exported a class but not a default class, then I'd have to declare
`import {Backend as Client} from './client';`